go.mod

```diff
@@ -289,6 +288,8 @@ require (
 
 require github.com/grafana/gofpdf v0.0.0-20231002120153-857cc45be447 // @grafana/sharing-squad
 
+require github.com/grafana/pyroscope/api v0.2.0
```

Comment:
what is up with this change?

Reply:
Added `@grafana/observability-traces-and-profiling` as the owner.
```
❯ go mod why -m github.com/grafana/pyroscope/api
# github.com/grafana/pyroscope/api
github.com/grafana/grafana/pkg/tsdb/grafana-pyroscope-datasource
github.com/grafana/pyroscope/api/gen/proto/go/querier/v1
```